doc/sphinx/userguide/faq.md

```diff
@@ -1,12 +1,227 @@
+
+See our documentation on [](sec-debugging-chemkin). If you're encountering an issue not
+described there, please post a question on the
+[Cantera Users' Group](https://groups.google.com/g/cantera-users).
+:::
+
+## Thermodynamics & Equilibrium
+
+:::{dropdown} Why don't the enthalpy and internal energy values calculated by Cantera match data from other sources (CoolProp / NIST / steam tables)? Why does my mixture have a negative enthalpy?
+:color: primary
+
+Reference states in thermodynamics are arbitrary, since the observable quantities of
+interest (work done, heat transferred) do not depend on absolute quantities but on
+differences between states. Different data sources use different conventions for the
+reference state. Cantera uses the convention commonly used in reaction thermodynamics,
+where pure elements in their natural state (for example, oxygen as O₂ gas and carbon as
+graphite) have zero enthalpy at a temperature of 298.15 K and a pressure of 1 atm. This
+convention provides a consistent treatment for the change in enthalpy associated with
+chemical reactions. In contrast, it is common for tabulated data for a single pure
+substance to use a different reference state, such as CO₂ where the enthalpy may be
+referenced to the saturated liquid at 273.15 K.
+:::
+
+:::{dropdown} Why doesn't the heat capacity calculated by Cantera match what I get from CEA?
+:color: primary
+
+Cantera’s definition of $c_p$ is the partial derivative of enthalpy with respect to
+temperature, with pressure and composition held constant. The value returned by CEA is
+the derivative while holding the system at chemical equilibrium. The former definition
+is the one that appears in governing equations for multi-species systems, such as
+[well-stirred reactors](/reference/reactors/ideal-gas-constant-pressure-reactor). While
+the latter property is of interest in certain physical situations, it is of somewhat
+limited use computationally, since it is *only* defined for mixtures at equilibrium.
+
+You can use Cantera to compute a value like what’s returned by CEA by computing $dh/dT$
+using a finite difference method where the mixture is equilibrated at constant $T$ and
+$P$ at each point before calculating the enthalpy.
+[`sound_speed.py`](/examples/python/thermo/sound_speed) presents a related example.
+
+CEA provides a value consistent with Cantera's definition which it calls the $c_p$ "with
+frozen reactions."
+:::
+
+:::{dropdown} Why doesn't changing reactions affect the equilibrium state calculated by Cantera? How does Cantera find the equilibrium state even when no reactions are defined?
+:color: primary
+Chemical equilibrium is not necessarily defined in terms of any reactions. For whatever
+set of species are allowed in a phase or multi-phase mixture, it is defined as the
+composition that minimizes the Gibbs free energy (for equilibrium at constant
+temperature and pressure). This composition may be different from what you would get,
```

Comment:
Since we're calling out T-P here, it might leave a question of, what about p-h or any other valid combination? Maybe saying something like "minimizes a thermodynamic state variable independent of which reactions are defined"?